__tests__/servertesting.test.js

```diff
@@ -41,12 +44,45 @@ describe("/get/api ", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then((data) => {
-        data = JSON.parse(data.text);
-        console.log(data);
-        expect(data).toHaveProperty("GET /api");
-        expect(data).toHaveProperty("GET /api/topics");
-        expect(data).toHaveProperty("GET /api/articles");
+      .then(({ body }) => {
+        expect(body).toEqual(data);
+      });
+  });
+});
+describe("/get/api/articles/:article_id ", () => {
+  it("200: Should return an article object selected by a parametric endpoint, with apropriate keys", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        body = body.articles;
+        expect(body.length > 0);
+        body.forEach((body) => {
+          expect(body).toHaveProperty("author");
+          expect(body).toHaveProperty("title");
+          expect(body).toHaveProperty("article_id");
+          expect(body).toHaveProperty("body");
+          expect(body).toHaveProperty("topic");
+          expect(body).toHaveProperty("created_at");
+          expect(body).toHaveProperty("votes");
+          expect(body).toHaveProperty("article_img_url");
```

Comment:
It's definitely a necessary part of this test to check for these properties to exist, but ideally we'd also be making sure they have the right value. Since we're requesting a single article with an exact Id, we can make assertions that know exactly what the value should be.
I would recommend either including a second argument for toHaveProperty (which is reserved for what the value should be), or using a matcher like .toMatchObject and specifying what the article should look like